x86_symbolic/src/Data/Macaw/X86/Crucible.hs

```diff
@@ -429,13 +468,38 @@ pureSem sym fn = do
 
     M.VExtractF128 {} -> error "VExtractF128"
 
+    M.CLMul x y ->
+      do let f = fnClMul (symFuns sym)
+             s = symIface sym
+         src1 <- toValBV s x
+         src2 <- toValBV s y
+         let ps = extend (extend empty src1) src2
+         llvmPointer_bv s =<< applySymFn s f ps
+
+    M.AESNI_AESEnc x y -> pureSemAESNI sym fnAesEnc x y
+    M.AESNI_AESEncLast x y -> pureSemAESNI sym fnAesEncLast x y
+    M.AESNI_AESDec x y -> pureSemAESNI sym fnAesDecLast x y
```

Comment:
Should this just be `fnAesDec`?

Reply:
Good catch - apparently I'm cursed to make these sorts of copy-paste errors twice over, as 0ff303d shows